E2E: Rename & move clearTextArea to correct spot

WunderBart opened the pull request:
#### Changes proposed in this Pull Request

The `clearTextArea` helper isn't actually for clearing `textarea` elements, as it checks for `innerText` of an element. We're using it only for clearing out paragraph block content in the GutenbergEditorComponent, so I moved it there and renamed to `clearText`.

#### Testing instructions

All tests should pass.

WunderBart committed:
Remove obsolete clearTextArea helper

noahtallen reviewed (approved):
Makes sense to me, assuming tests pass

WunderBart committed:
Don't use setWhenSettable for filling out paragraphs
...because they automatically split into multiple paragraphs on newline, so the value validation in setWhenSettable fails.

WunderBart committed:
Move clearText to Gutenberg Component
Seems like we need it after all. It's not for clearing textareas as it seemes to be, though, so I'm moving it to correct spot.

WunderBart commented:
Had to change direction a bit with this one - would appreciate if you could take another look, @noahtallen! 🙏

noahtallen commented:
LGTM! 🚀
